python: Fix code lints

cathay4t opened the pull request:
Bump pylint, black, flake8 to latest version.

Run the python lint check in CI host directly to speed it up.

ffmancera commented on .github/workflows/main.yml:
```diff
@@ -76,21 +76,16 @@ jobs:
   integ:
     runs-on: ubuntu-latest
-    needs: [rust_lint, py_lint, rpm_build]
+    needs: [rust_lint, py_lint, rpm_build, py_lint]
```
just a nit, but `py_lint` is duplicated here. other than that it looks good!

cathay4t committed:
python: Fix code lints

Bump the pylint, black, flake8 to latest version.

Run the python lint check in CI host directly to speed it up.

Signed-off-by: Gris Ge <fge@redhat.com>
